site/en/blog/cascade-layers/index.md

```diff
@@ -0,0 +1,253 @@
+
+**UA normal < Local User @layer < Local User normal < Author @layers < Author normal < Author !important < Author @layer !important < Local User !important < UA !important**
+
+You may notice here that `@layer !important` styles are inverted. Instead of being *less* specific than non-layered (normal) styles, they become *more* specific. This is because of how `!important` works in the cascade: it breaks the normal cascading in your stylesheets and reverses the normal specificity values.
+
+### Nested layers
+Layers can also be nested within other layers. The following example comes from the [Cascade Layers explainer](https://css.oddbird.net/layers/explainer/) from Miriam Suzanne:
+
+```css
+@layer default {
+  p { max-width: 70ch; }
+}
+
+@layer framework {
+  @layer default {
+    p { margin-block: 0.75em; }
+  }
+
+  p { margin-bottom: 1em; }
+}
+```
+
+In the above code snippet, you can access `framework.default`, using a `.` as a signifier of the `default` layer being nested within `framework`. You can also write this in a more shorthand format:
+
+```css
+@layer.default {
+  p { margin-block: 0.75em }
+}
+```
+
+The resulting layers and layer-order are:
+
+- default
+- `framework.default`
+- `framework` unlayered
+- unlayered
+
+{% Aside %}
+*Shadow DOM*: Layers encapsulated within the shadow DOM do not cross the Shadow DOM boundary. Therefore, identically named layers in the shadow DOM have no impact on the order of layers in the light DOM and vice versa.
+{% endAside %}
+
+## Things to look out for
+
+Cascade layers can be great if you use them correctly, but they can also create additional confusion and unexpected results. Look out for the following when working with cascade layers:
+
+### Rule 1: Don’t use `@layer` for scoping
+
+Cascade layers do not solve scoping. If you have  a CSS file with an @layer, say `card.css` and want to style all of the links within card, do *NOT* write styles like:
+
+```css
+a {
+  …
+}
+```
+
+This will lead to all of the `a` tags in your file getting this override. It’s still important to **scope** your styles properly:
+
+```css
+.card a {
+  …
+}
+```
+
+### Rule 2: cascade layers are ordered behind non-layered CSS
+
+It’s important to note that a layered CSS file will *not* override non-layered CSS. This was an intentional decision to make it easier to introduce layers in a more sensible way to work with your existing codebase. Using a `reset.css` file, for example, is a good starting point and use case for cascade layers. 
+
+### Rule 3: `!important` inverts cascade specificity
+
+While layered styles are less specific than unlayered styles in general, using `!important` reverses this.
+
+> Layers are less specific than unlayered styles *unless* they are `!important` styles within a layer.
+
+In that case, the `!important` styles invert their specificity. The diagram above shows this for reference: `Author @layers < Author normal < Author !important < Author @layer !important`.
```

Comment:
can we write this one out too, rather than using `<`.

Reply:
this feels easier to read than `Author @layers have less precedence than Author normal which have less precedence than Author !important which have less precedence than Author @layer !important` but I'm happy to use either